microsoft/botframework-solutions
# Fix email detail read and localization issue

## Description
Fix email detail read and localization issue

## Related Issue
Email template parts are hardcoded instead of localized #163

## Testing Steps
1. Make a zh-cn bot and check the localization string.
2. Use voice input to make sure details are all read out.
